docs/setconsolescreenbufferinfoex.md

```diff
@@ -59,6 +59,11 @@ If the function succeeds, the return value is nonzero.
 
 If the function fails, the return value is zero. To get extended error information, call [**GetLastError**](https://msdn.microsoft.com/library/windows/desktop/ms679360).
 
+## Remarks
+
+> [!TIP]
+> This API has a partial **[virtual terminal](console-virtual-terminal-sequences)** equivalent. **[Cursor positioning buffer](console-virtual-terminal-sequences#cursor-positioning)** and **[text attributes](console-virtual-terminal-sequences#text-formatting)** have specific sequence equivalents. The color table is not configurable, but **[extended colors](console-virtual-terminal-sequences#extended-colors)** are available beyond what is normally available through **[console functions](console-functions.md)**. Popup attributes have no equivalent as popup menus are the responsibility of the command-line client application in the **virtual terminal** world. Finally, the size of the window and the full screen status are considered privileges owned by the user in the **virtual terminal** world have no equivalent sequence.
```

Comment:
The color table is infact configurable! Through _yet more VT!_ :smile:

Reply:
I'm pretty sure we didn't document wtf that VT is in the VT sequences doc and I don't have time to do that, so I'm saying it isn't.